src/lib/eZ/RichText/Resources/stylesheets/xhtml5/edit/docbook.xsl

```diff
@@ -747,4 +747,22 @@
     <xsl:value-of select="translate( substring-before( substring-after( concat( substring-after( $style, $property ), ';' ), ':' ), ';' ), ' ', '' )"/>
   </xsl:template>
 
+  <!-- Some fallbacks to handle translating inline formatting in span tags for copy&paste and import use cases -->
+  <xsl:template match="ezxhtml5:span[not(@data-ezelement) and contains(@style,'bold')]">
```

Comment:
Not a huge fan of that.
POV ping @dew326: Can we rely on a CSS style containing "bold" phrase to always convert entire `<span>` as `<strong>` for OE format?

Reply:
I'll let @dew326 confirm. But afaik we don't use it, it would come from copy&paste or import cases, and in those cases there might be lots of styles at the same time, so in this case as the test shows the last one wins. I'm not aiming for perfection here just some more flexibility in Html5edit input parsing.

From what I could see `bold` and `bolder` are the only words in CSS terminology that will match this.

Reply:
From the AE perspective, it doesn't matter. We are not using this and AE will never produce such format.

Reply:
@alongosz WDYT? :)